Source/Core/Core/PowerPC/JitArm64/JitArm64_FloatingPoint.cpp

```diff
@@ -345,78 +345,73 @@ void JitArm64::fcmpx(UGeckoInstruction inst)
 	JITDISABLE(bJITFloatingPointOff);
 
 	u32 a = inst.FA, b = inst.FB;
-	int crf = inst.CRFD;
 
 	ARM64Reg VA = fpr.R(a, REG_IS_LOADED);
 	ARM64Reg VB = fpr.R(b, REG_IS_LOADED);
 
-	ARM64Reg WA = gpr.GetReg();
-	ARM64Reg XA = EncodeRegTo64(WA);
-
-	FixupBranch pNaN1, pNaN2, pNaN3, pLesser, pGreater;
-	FixupBranch continue1, continue2, continue3;
-	ORR(XA, ZR, 32, 0, true);
+	FixupBranch pLesser, pGreater;
 
 	if (a != b)
 	{
 		m_float_emit.FCMP(EncodeRegToDouble(VA), EncodeRegToDouble(VA));
 
-		// if (B != B) or (A != A), goto NaN's jump target
-		pNaN1 = B(CC_NEQ);
+		// if (B != B) or (A != A), Fall back to interpreter
+		if (CC_NEQ)
```

Comment:
None taken, cause you are very much right. This is a learning process for me as well, never done any backend work before, and I kinda knew it was too early for me to get involved with JIT work but I thought I'd give it a shot anyways. I'll stick to frontend work for now and make my focus implementing/porting desktop functions to the Android version.
The Arm64 is a backend and Android is a frontend, right? or have I got one more thing wrong as well ;)

Maybe I'll try to get back into working on the JIT, but not for a while at least, lots of stuff I have to get a better grasp on first. Thanks for the support throughout my attempt at JIT work, though.